pkg/oc/util/clientcmd/resolve.go

```diff
@@ -53,6 +53,8 @@ func resolveImagePullSpec(imageClient imageclient.ImageInterface, source, name,
 		return "", fmt.Errorf("unable to resolve %s %q", source, name)
 	}
 
+	// TODO(juanvallejo): This helper might need to be copied into the `oc` tree in order
+	// to snip this dependency on the internal image api tree
```

Comment:
This should be replaced with https://github.com/openshift/library-go/blob/a7d809ee41d6ca6720192824ee54ed8f19c3cfac/pkg/image/reference/reference.go#L32, if you look under the covers of `imageapi.ParseDockerImageReference` it call just that function.

Reply:
That's great, didn't know about this helper. Will sweep remaining uses of the imageutil helper as well